src/runtime/Makefile

```diff
@@ -323,6 +333,18 @@ ifneq (,$(QEMUCMD))
 
     CONFIGS += $(CONFIG_QEMU_SEV)
 
+    CONFIG_FILE_QEMU_SNP = configuration-qemu-snp.toml
+	CONFIG_QEMU_SNP = config/$(CONFIG_FILE_QEMU_SNP)
+    CONFIG_QEMU_SNP_IN = $(CONFIG_QEMU_SNP).in
+
+    CONFIG_PATH_QEMU_SNP = $(abspath $(CONFDIR)/$(CONFIG_FILE_QEMU_SNP))
+	CONFIG_PATHS += $(CONFIG_PATH_QEMU_SNP)
+
+	SYSCONFIG_QEMU_SNP = $(abspath $(SYSCONFDIR)/$(CONFIG_FILE_QEMU_SNP))
+    SYSCONFIG_PATHS += $(SYSCONFIG_QEMU_SNP)
+
+	CONFIGS += $(CONFIG_QEMU_SNP)
```

Comment:
The indentation is inconsistent (tabs mixed with spaces). You want spaces on all lines.